simuleval/utils/slurm.py

```diff
@@ -65,10 +80,33 @@ def submit_slurm_job(config_dict: Optional[Dict] = None) -> None:
             r"[^\"'\s]+\.py", f"{os.path.abspath(args.output)}/agent.py", command
         ).strip()
 
+    sweep_command = ""
+    sbatch_job_array_head = ""
```

Comment:
not blocking, but submitit has a nice interface for job arrays (https://github.com/facebookincubator/submitit/blob/main/docs/examples.md#job-arrays), if we want to migrate soon

Reply:
Thanks! Yeah will definitely visit this later. I admit that the current approach is pretty hacky ....